BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/Views/SkontoWithDiscountPriceView.swift

```diff
@@ -33,7 +33,6 @@ class SkontoWithDiscountPriceView: UIView {
 
     private func setupView() {
         translatesAutoresizingMaskIntoConstraints = false
-        backgroundColor = .giniColorScheme().bg.inputUnfocused.uiColor()
```

Comment:
[clarification] Is this ok that we removed this?

Reply:
yes, it was extra and this background is not needed